content/TruenasApps/StableApps/PlexApp.md

```diff
@@ -0,0 +1,263 @@
+
+After signing in, Plex steps you through several initial media server configuration screens if this is a new account, or shows your default Plex media screen for your existing configured account.
+
+## Understanding App Installation Wizard Settings
+The following section provides more detailed explanations of the settings in each section of the **Install** installation wizard.
+
+### Application Name Settings
+
+{{< include file="/static/includes/apps/AppsWizardAppNameAndVersion.md" >}}
+
+### Plex Configuration Settings
+Plex configuration settings include setting up the server timezone, authentication to the Plex account, location of the Plex container image, local network settings, and adding devices or additional environment variables to apply to the container.
+
+{{< trueimage src="/images/SCALE/Apps/InstallPlexConfig1.png" alt="Instll Plex Configuration Settings" id="Install Plex Configuration Settings" >}}
+
+{{< include file="/static/includes/apps/AppsInstallWizardTimezoneSetting.md" >}}
+
+Authentication to the Plex account is accomplished through a [claim token provided by Plex](https://www.plex.tv/claim/).
+While signed into your Plex account, copy the token provided by Plex into the **Claim Token** field.
+The Plex app does not deploy if you do not include this token.
+
+Accept the default value in **Image** to use the container image for the TrueNAS app.
+
+#### Setting Up Local Network
+TrueNAS shows the default IP addresses detected for your system.
+If these address fields are not shown in the wizard, your network is considered to be on the external network.
+
+Either accept the default values or enter the IP addresses for local network connections (Ethernet or WiFi routers) you want in your Plex local network.
+These addresses define how Plex interacts with devices and services on your network and can optimize how your Plex media server communicates with devices in your home.
+
+Specified addresses are considered to be on the local network when enforcing bandwidth restrictions.
+If left blank, only the subnet for the server is considered to be on your local network.
+When the host network is not enabled, the server subnet is the network for Docker.
+Therefore, all connections from other clients are considered to be on the external network.
+If set, all other IP addresses are considered to be external to your local network.
+
+#### Adding Devices
+Plex allows you to add and manage devices allowed to access your media server through the local network and remotely from external network connections.
+Click **Add** to the right of **Devices** in the **Install Plex** wizard for each device to add.
+
+{{< trueimage src="/images/SCALE/Apps/InstallPlexConfigAddDevice.png" alt="Install Plex Add Device" id="Install Plex Add Device" >}}
+
+Enter the name of the device in the **Host Device** and in the **Container Device** fields. For example, device */dev/dvb*.
```

Comment:
Something seems off about the description here - is this about passing system devices directly through to the container or about permissions for clients to access the Plex server over the network?